boards/arm64/rcar_h3ulcb_ca57/Kconfig.board

```diff
@@ -4,4 +4,3 @@
 config BOARD_RCAR_H3ULCB_CA57
 	bool "Renesas H3ULCB"
 	depends on SOC_R8A7795
-	select ARM64
```

Comment:
ha, it is called development process, when you think that smth is needed and that you have opposite thoughts, because you have found out that it is already selected by other dependencies
I'll think what I can do with this